docs/snippets/common/storybook-test-runner-error.js.mdx

```diff
@@ -0,0 +1,3 @@
+```shell
+Timeout - Async callback was not invoked within the 15000 ms timeout specified by jest.setTimeout
```

Comment:
Do we need all this stuff in snippets? What's the rationale?

I would suggest that we only snippetize the stuff that we plan to translate into different languages/frameworks. Standalone snippets are harder to review and contextualize.

But maybe there is some added benefit that I'm missing?